### PR TITLE
Corrections for 2.3 character set rules

### PR DESCRIPTION
**Pull request by Dave-Allured**
* Correct the paraphrasing of the NUG.
* Add link.

### CODE REVIEW

File: ch02.adoc [hunk 1]
```diff
@@ -54,8 +54,8 @@ The examples in this document that use string-valued variables alternate between
 === Naming Conventions
 
 Variable, dimension, attribute and group names should begin with a letter and be composed of letters, digits, and underscores.
```
Comment (larsbarring) [inline ch02.adoc:56]: Also clarify what is meant by  "a letter".

Comment (Dave-Allured) [reply ch02.adoc:56]: @larsbarring, please submit that as a separate PR.  Thank you.

**Comment by Dave-Allured** [comment]
This PR #465 is in violation of CF governance, paragraph 1 of [Rules for CF Conventions Changes](http://cfconventions.org/rules.html), which requires a Github issue to discuss all changes.  My apologies.  I withdraw this PR.  I will submit a new issue and try to follow correct protocol hereafter.

**Comment by Dave-Allured** [comment]
After thinking it over, I am cancelling this proposal.  Interested persons are encouraged to copy any of these suggestions into their own proposals, without credit.

**Pull request closed without merging**